CMakeLists.txt

```diff
@@ -128,9 +227,7 @@ macro(nbl_adjust_flags)
 				"$<$<CONFIG:RELEASE>:-ffast-math>"
 			)
 		endif()
-		if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
-			add_compile_options("$<$<CONFIG:DEBUG>:-ggdb3>")
-		endif()
```

Comment:
why!?

without this GDB gets lost in the debug info and cant see file lines!

Reply:
hm, worked fine for me with only `-g` while debugging on linux

Reply:
but ofc ill bring it back